### DISCUSSION TITLE
通常世界の完成度

### DISCUSSION THREAD

**Discussion by anminmakura** [TUSB/TheUnusualSkyBlock#69] [エリア・建築の完成度・開発状況]
- [x] 飛空島用の駅島の製作
_現状だと初期地点の交易島の駅しかない。_
_残り２つ追加予定。_
- [x] 各島にちゃんとした名前付ける
_新規の島が多く、名前が付けられていない島が多い。_

他要素はほぼ完成しているが、1.16環境になり次第1.16のブロックを散りばめたい。

**Comment by anminmakura** [comment]
![エンチャント島中身を1 16化](https://user-images.githubusercontent.com/39670701/99180443-b5330080-2769-11eb-919a-8e0b6ffbb171.png)
れんと氏がエンチャント島中身を1.16化。

**Comment by anminmakura** [comment]
しょぼりんぬ氏が作業台島を製図台島にリメイク。
尚しょぼ氏曰く、外装が未完成なので作れる人を募集中とのこと。
![製図台01](https://user-images.githubusercontent.com/39670701/100337149-619e9d80-301a-11eb-9337-92ed90486f5c.png)
![製図台02](https://user-images.githubusercontent.com/39670701/100337161-64998e00-301a-11eb-9272-af5a091d7fc5.png)
![製図台03](https://user-images.githubusercontent.com/39670701/100337177-682d1500-301a-11eb-8baf-de6ec7ec3b3a.png)

**Comment by huyuyori** [comment]
![2020-11-28_22 18 56](https://user-images.githubusercontent.com/73624149/100516521-db27be80-31c7-11eb-82ab-c5e4b8e12fce.png)
内装完成

**Comment by anminmakura** [comment]
## 飛空島(旧名：スキル設定所)にTPする装置がある島を2箇所建築
TP装置は3箇所あり、１つは交易島にあるので、残り2つは新規で製作予定。

正確な座標は追々書き込む予定ではあるが、現状としてこの2箇所周辺に建築予定。

### ・クラウディア入り口

### ・テーブルマウンテン入り口

以下何故この場所が選ばれたのかの理由

### クラウディア入り口が選ばれた理由
・上位の島の実力を味わってもらえる。
・高難易度でありながらバランス壊すようなアイテムの配置はない。
・最初からエンチャント施設いける(尚Lv1～5でクリアできればの話)
・上に移動する手間がなくなる(仕掛けに翻弄されなければ)
・遠くにいく手間が減る(仕掛けにry

### テーブルマウンテン入り口が選ばれた理由

・12vと違い、即テーブルには進入できない。
・近くには資源があるがぶっ壊れ系のものはない。

**Comment by reiOmoti** [comment]
豪邸島1.16化
![2020-12-24_13 51 35](https://user-images.githubusercontent.com/76546701/103062138-b40ab400-45f0-11eb-9c0b-ab160862c5d2.png)

**Comment by anminmakura** [comment]
現状の通常世界の全体図まとめ

![通常世界全体図](https://user-images.githubusercontent.com/39670701/109440669-668a4780-7a76-11eb-90bd-8e896c5af5eb.png)

### A:通常世界初期地点島


### B:地下世界ゲート島


### C1:飛空島ゲート(交易島駅)


### C2:飛空島ゲート(エンチャント島前、牢獄の駅)


### C3:飛空島ゲート(テーブルマウンテンゲート前、水の駅)


### D:テーブルマウンテンゲート島


### E:ガリバーランドゲート島


### F:ネザーゲート島


### G:クラウディアゲート島


### H:トカルトゲート島


### I:ネザーアスレ島


### J:旧テーブルマウンテンゲート島の位置にバベルのゲート建設予定


### K:神木島地下に虚数領域ゲート建設予定


### L:追憶の海ゲート(砂漠ゲート)


### X:旧デバッグルーム


### Y:0 0 0の初期地点

**Comment by anminmakura** [comment]
01 通常世界初期地点 -3 3 -621

02 砂利島 -16 5 -621

03 羊島 -3 7 -598

04 ひまわり島(元荒れ地島) -22 6 -593

05 丸土島 -32 38 -608

06 交易島(飛空島ゲート) -45 11 -610

07 卵島 -28 18 -644

08 月と太陽の像 15 13 -621

09 砂島 -7 8 -644

10 火炎放射島 4 3 -651

11 時計島 47 12 -602

12 ガスト島 7 7 -578

13 シミ島 27 10 -648

14 脅威島 39 5 -562

15 通常世界 ネザーアスレゲート -14 37 -515

16 イカ島 -37 61 -570

17 舵島 62 31 -452

18 月島 3 78 -540

19 通常世界 バベルゲート -94 17 -560

20 遺跡島 -121 16 -599

21 灯台島 -57 0 -649

22 通常世界 地下世界ゲート -32 9 -685

23 エメラルド島 58 14 -661

24 水島 67 9 -628

25 氷島 60 5 -689

26 墓場島 16 10 -676

27 クリーパー島 18 40 -658

28 スティーブ島 -48 64 -675

29 燃える愛の島 -4 91 -633

30 ずたぼろ亭 -46 85 -621

31 かぼちゃ島 35 73 -597

32 スイカ島 12 130 -588

33 五重の塔 66 27 -546

34 蜘蛛島 107 47 -568

35 本の神殿 180 22 -559

36 通常世界 テーブルマウンテンゲート 134 52 -476

37 セキュリティーポール島(元かわいい顔の島) 122 36 -621

38 飛空島ゲート(テーブルマウンテンゲート前、水の駅) 115 34 -528

39 通常世界 ガリバーランドゲート 84 24 -679

40 逆さ家 67 51 -677

41 ケーキ島 98 126 -645

42 かまど島 88 22 -728

43 たけのこ島 163 63 -762

44 通常世界 ネザーゲート 119 71 -701

45 大釜とホッパー島 165 81 -698

46 豪邸島 141 56 -651

47 オアシス島 -107 22 -704

48 製図台島(元作業台島) -49 52 -737

49 海賊船島 28 17 -783

50 スケルトンとウィザースケルトンの島 116 126 -588

51 エンダーマンとシュルカーとエンダーマイトの島 81 134 -549

52 神木島(通常世界 虚数領域ゲート) 212 114 -479

53 通常世界 トカルト・コルデゲート 200 145 -644

54 銭湯島 202 130 -747

55 エンダーチェスト島 82 110 -719

56 エンチャント島 -25 113 -665

57 処刑塔 19 137 -508

58 魔女の館島(魔女島) -78 177 -555

59 廃病院島 -115 185 -674

60 深淵島(養豚場) -95 179 -775

61 通常世界 クラウディアゲート 6 150 -762

62 死島(なれんじ城) 104 176 -827

63 隔離所(れい城) -179 94 -761

64 飛空島ゲート(エンチャント島前、牢獄の駅) -64 121 -699

65 通常世界 追憶の海(砂漠) 78 197 -587

67 通常世界 エンドゲート 21 243 -664
